server/instance_cloud.go

```diff
@@ -132,7 +133,19 @@ func (jci jiraCloudInstance) GetJIRAClient(jiraUser JIRAUser) (*jira.Client, err
 }
 
 // Creates a client for acting on behalf of a user
-func (jci jiraCloudInstance) getJIRAClientForUser(jiraUser JIRAUser) (*jira.Client, *http.Client, error) {
+func (jci jiraCloudInstance) getJIRAClientForUser(jiraUser JIRAUser) (*jira.Client, error) {
```

Comment:
Why are we caching the user client, I thought we use the `getJIRAClientForServer` while processing the webhooks and cache it's value?

Reply:
Wow that's a big goof :frowning_face: , this is in the wrong function. Thank you for catching this